Client/Frontend/Browser/FirefoxTabContentBlocker.swift

```diff
@@ -8,8 +8,10 @@ import Shared
 struct ContentBlockingConfig {
     struct Prefs {
         static let StrengthKey = "prefkey.trackingprotection.strength"
```

Comment:
Is this still in use?

Reply:
Yes. It will be removed in scope of #273.